src/checkbox/internal.tsx

```diff
@@ -56,6 +67,20 @@ const InternalCheckbox = React.forwardRef<CheckboxProps.Ref, InternalProps>(
 
     const { tabIndex } = useSingleTabStopNavigation(checkboxRef, { tabIndex: explicitTabIndex });
 
+    const analyticsMetadata: GeneratedAnalyticsMetadataFragment = {};
+    const analyticsComponentMetadata: GeneratedAnalyticsMetadataCheckboxComponent = {
+      name: 'awsui.Checkbox',
+      label: '',
+    };
+    if (__injectAnalyticsComponentMetadata) {
+      analyticsMetadata.component = analyticsComponentMetadata;
+    }
+    if (!disabled && !readOnly) {
+      analyticsMetadata.detail = {
+        selected: `${!checked}`,
```

Comment:
why selected is the opposite of checked?

Reply:
Because we surface the future state